locale/en/about/community.md

```diff
@@ -0,0 +1,53 @@
+ 2. A **Collaborator** is a contributor who has been given write access to the repository
+ 3. An **Observer** is any individual who has requested or been requested to attend a CommComm meeting. It is also the first step to becoming a Member.
+ 4. A **Member** is a collaborator with voting rights who has met the requirements of participation and voted in by the CommComm voting process.
+
+For the current list of Community Committee members, see the project's [README.md](https://github.com/nodejs/community-committee).
+
+## Collaborators
+
+It is the mission of CommComm to further build out the Node.js Community. If you're reading this, you're already a part of that community – and as a part of the Node.js Community, we'd love to have your help!
+
+The [nodejs/community-committee](https://github.com/nodejs/community-committee) GitHub repository is a great place to start. Check out the [issues labeled "Good first issue"](https://github.com/nodejs/community-committee/labels/good%20first%20issue) to see where we're looking for help. If you have your own ideas on how we could engage and build the community, feel free to create your own issue.
+
+If you're interested in participating in the Community Committee as a member, you should create an issue asking to be an Observer in our next Community Committee meeting. You can find a great example of such an issue [here](https://github.com/nodejs/community-committee/issues/142).
+
+Before diving in though, please be sure to read the [Collaborator Guide](https://github.com/nodejs/community-committee/blob/master/COLLABORATOR_GUIDE.md).
+
+## Membership
+
+To become a member of the Community Committee, start participating! You must:
+
+ 1. Attend the bi-weekly meetings, investigate issues tagged as good first issue, file issues and pull requests, and provide insight via GitHub.
+ 2. Request to become an Observer by filing an issue. Once added as an Observer to meetings, we will track attendance and participation for 3 months, as according to our governance guidelines.
+ 3. When you meet the minimum attendance and participation expectations, the CommComm will vote to add you as a member.
+
+Membership is for 6 months. The group will ask on a regular basis if the expiring members would like to stay on. A member just needs to reply to renew. There is no fixed size of the CommComm. However, the expected target is between 9 and 12. You can read more about membership, and other administrative details, in our [Governance Guide](https://github.com/nodejs/community-committee/blob/master/GOVERNANCE.md).
+
+Regular CommComm meetings are held bi-monthly in a Zoom video conference, and broadcast live to the public on YouTube. Meeting announcements and agendas are posted before the meeting begins in the organization's [GitHub issues](https://github.com/nodejs/community-committee/issues). Any community member or contributor can ask that something be added to the next meeting's agenda by logging a GitHub Issue.
```

Comment:
Is there a better way for newbies to discover when/how meetings are held? Right now its "keep an eye out on the issues list for the announcement". Not the easiest way for those not plugged in yet to engage.

Reply:
Maybe advertise the timing of the meetings? Although I realize it's subject to change. But we know it's definitely on thursday at a precise UTC hour?

Reply:
One way to keep on top of meetings (not just CommComm, but all Node.js meetings):
- Subscribe to the Node.js Foundation YouTube channel (https://www.youtube.com/channel/UCQPYJluYC_sn_Qz_XE-YbTQ) 
- Turn on Channel Notifications: 
![image](https://user-images.githubusercontent.com/502396/32744109-8c6c7982-c87c-11e7-81ad-3c1c79330e77.png)

That will alert anyone interested in when a livestream is going online, and (if the stream is scheduled beforehand, something @rachelnicole and I need to work on doing) get notice well before the stream is actually live 👍

Reply:
It should also be on the standard calendar, and its at a regular time so unless it changes it should be easy to know when the next one is: https://nodejs.org/calendar